modules/ROOT/pages/a11ychecker.adoc

```diff
@@ -399,35 +404,39 @@ console.log(issues);
 // example result
 [
   {
+    "contentID": "<div id=\"accessibility-issue__contentID\"><span>Text:&nbsp;</span><span>\"H5\"</span></div>",
+    "description": "Headings must be applied in sequential order. For example: Heading 1 should be followed by Heading 2, not Heading 3.",
+    "element": [object],  // The element value contains the DOM element (such as <h4>).
```

Comment:
I wasn't too sure on this `[object]`, we already clearly labeled this as example, why are we not just use `element: h4` here